jenkins/Jenkinsfile-release

```diff
@@ -45,11 +40,11 @@ node {
         
             sh 'find . -type f -regex ".*/target/symphony-api-client-java-.*.jar" -exec gpg --batch --passphrase $PASSPHRASE -ab {} \\;'
             sh 'ls -la ./symphony-bdk-legacy/symphony-api-client-java/target'
-            sh '''
-                cd ./symphony-bdk-legacy/symphony-api-client-java
-                pwd
-                mvn --batch-mode --settings ../../jenkins/config-release/settings-release.xml -Dgpg.passphrase=$PASSPHRASE -Drepo.id=ossrh -Drepo.username=symphony-platform-solutions -Drepo.password=$PASSPHRASE clean deploy
-            '''
+            sh 'mvn --batch-mode --settings ../../jenkins/config-release/settings-release.xml -P release,$PARAM_REPO_ID ' +
+                    '-Dgpg.passphrase=$PASSPHRASE ' +
+                    '-Drepo.id=$PARAM_REPO_ID -Drepo.username=symphony-platform-solutions -Drepo.password=$PASSPHRASE ' +
+                    '-Dsym.repo.url=$PARAM_SYM_REPO_URL ' +
+                    'clean deploy'
```

Comment:
will the clean deploy work of Maven Central/ossrh? I thought we had to use the nexus plugin

Reply:
Yes, according to https://help.sonatype.com/repomanager2/staging-releases/configuring-your-project-for-deployment#ConfiguringYourProjectforDeployment-DeploymentwiththeNexusStagingMavenPlugin :

> This configuration works only in Maven 3 and automatically replaces the deploy goal invocation of the Maven Deploy plugin in the deploy Maven life cycle phase with the deploy goal invocation of the Nexus staging Maven plugin.

Reply:
👍 

Do you think sym.repo.url is going to change? To me it is unlikely so I would perhaps not make it a parameter of the job

Perhaps we can already start creating the jobs to check that it works (using your PR's branch)

Reply:
> Do you think sym.repo.url is going to change? To me it is unlikely so I would perhaps not make it a parameter of the job

As this project is open-source, I prefer to keep this kind of info outside of the codebase

Reply:
Indeed 👍